docs/dns-usage.rst

```diff
@@ -0,0 +1,184 @@
+Using the API
+=============
+
+Authorization / Configuration
```

Comment:
1. Use authentication instead of authorization.
1. We can just link to http://gcloud-python.readthedocs.org/en/latest/gcloud-auth.html instead of writing anything
1. You can probably just talk about config